### DISCUSSION TITLE
How do I listen for application installation and uninstallation

### DISCUSSION THREAD

**Discussion by Ag0v0** [tauri-apps/tauri#5968] [Q&A]
I did not find any monitoring hooks about application installation and uninstallation in the documentation. How should I monitor them😀

**Comment by FabianLars** [comment]
There is no such thing. The only thing i can think of is modifying the WiX template: https://tauri.app/v1/guides/building/windows#customizing-the-installer to include scripts you want to run at un-/install time.
